app/assets/stylesheets/main.scss

```diff
@@ -335,5 +337,12 @@ body {
     border: unset;
   }
 }
-//
 
+.mentions__mention {
```

Comment:
I would recommend to use CSS naming convention here - "mentions-mention" . 
[Here](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) you can read about BEM naming convention you`ve used above

Reply:
Ok, thanks, I just used styles from mentions lib - https://github.com/signavio/react-mentions/blob/master/demo/src/examples/example.module.css